src/main/java/io/vertx/rabbitmq/impl/RabbitMQClientImpl.java

```diff
@@ -87,7 +107,11 @@ private static Connection newConnection(RabbitMQOptions config) throws IOExcepti
     cf.setNetworkRecoveryInterval(config.getNetworkRecoveryInterval());
     cf.setAutomaticRecoveryEnabled(config.isAutomaticRecoveryEnabled());
 
-
+    if(config.isSsl()) {   	
+    	SSLHelper sslHelper = new SSLHelper(config, config.getKeyCertOptions(), config.getTrustOptions());
```

Comment:
we should force on the config the `JdkSSLEngineOptions` to be used as SSL engine options in order to guarantee that he cast to `JdkSslContext` will be fine